wallet/src/actors/worker/methods.rs

```diff
@@ -1074,7 +1074,7 @@ impl Worker {
         wallet: types::SessionWallet,
         sink: types::DynamicSink,
     ) -> Result<()> {
-        log::info!("The node has changed its status into {:?}", status);
+        log::info!("The current node status is {:?}", status);
```

Comment:
We will see this every 15 seconds, should we move the log to debug?

```suggestion
        log::debug!("The current node status is {:?}", status);
```